### PR TITLE
Prepare 0.1.0 release, add release process

### PR DESCRIPTION
**Pull request by jameshalsall**

### CODE REVIEW

File: tools/docs/advanced/using-multiple-main-topics.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+```
+KAFKA_SOURCE_TOPICS=price,product
+KAFKA_RETRY_INTERVALS=120
+KAFKA_GROUP=algolia
+```
+
+would generate the following topic chains:
+
+1. `price` -> `retry1.algolia.price` (delay of 120 secs) -> `deadLetter.algolia.price`
+1. `product` -> `retry1.algolia.product` (delay of 120 secs) -> `deadLetter.algolia.product`
```
Comment (hgajjar) [inline tools/docs/advanced/using-multiple-main-topics.md:16]: 2.

File: tools/docs/releasing.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Releasing
+
+We keep a changelog, powered by [GitHub Changelog Generator].
+
+When ready to tag a release, make a new branch from the `master` branch for the changelog entries:
+1. Generate a `repo` scope token for use with the changelog generator: https://github.com/settings/tokens/new?description=GitHub%20Changelog%20Generator%20token
+1. Export it in your environment: `export CHANGELOG_GITHUB_TOKEN=...`
+1. Run the following docker command to generate the changelog, replacing `<nextReleaseTag>` with the version number you
```
Comment (hgajjar) [inline tools/docs/releasing.md:8]: wrong numbering

File: tools/docs/releasing.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+   wish to release:
+  ```bash
+  docker run -e CHANGELOG_GITHUB_TOKEN="$CHANGELOG_GITHUB_TOKEN" -it --rm -v "$(pwd)":/usr/local/src/your-app -v "$(pwd)/github-changelog-http-cache":/tmp/github-changelog-http-cache ferrarimarco/github-changelog-generator --user inviqa --project kafka-consumer-go --exclude-labels "duplicate,question,invalid,wontfix,skip-changelog" --release-branch master --future-release <nextReleaseTag>
+  ```
+1. Examine the generated `CHANGELOG.md`, verify the changes make sense.
+1. Commit the resulting `CHANGELOG.md`, push and raise a pull request with the label `skip-changelog`.
+1. Once merged, continue with the release process below.
```
Comment (hgajjar) [inline tools/docs/releasing.md:15]: same as above

File: tools/docs/releasing.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## Finalising a Release
+
+Once the `CHANGELOG.md` is in the branch you wish to release:
+
+1. Tag the release version with `git tag <releaseVersion>`
+1. Push the tag to the repository: `git push origin <releaseVersion>`
+1. Create a [new release] using the tag name and the contents of the changelog for that version.
```
Comment (hgajjar) [inline tools/docs/releasing.md:23]: same as above

File: tools/docs/advanced/using-multiple-main-topics.md (hunk 1 above)
Comment (jameshalsall) [reply tools/docs/advanced/using-multiple-main-topics.md:16]: this is automatically incremented when displayed, see https://github.com/inviqa/kafka-consumer-go/blob/392a6c7cfd6748aebade50107e8961cd0fe2dabd/tools/docs/advanced/using-multiple-main-topics.md

File: tools/docs/releasing.md (hunk 2 above)
Comment (jameshalsall) [reply tools/docs/releasing.md:8]: same here, see https://github.com/inviqa/kafka-consumer-go/blob/392a6c7cfd6748aebade50107e8961cd0fe2dabd/tools/docs/releasing.md

File: tools/docs/advanced/using-multiple-main-topics.md (hunk 1 above)
Comment (hgajjar) [reply tools/docs/advanced/using-multiple-main-topics.md:16]: oh

**Review by hgajjar — APPROVED** [review]

**Pull request merged**